extras/run.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+env TMPDIR="$XDG_RUNTIME_DIR/app/${FLATPAK_ID:-chat.revolt.RevoltDesktop}" zypak-wrapper /app/bin/revolt-desktop-1.0.6/revolt-desktop "$@"
```

Comment:
Why is TMPDIR here? What does it do? It should be documented in order to ease maintenance. I'd also prefer to create this file using the script source type.

Reply:
i will try that